packages/nimble-components/src/table/index.ts

```diff
@@ -243,6 +244,8 @@ export class Table<
         this.table = tanStackCreateTable(this.options);
         this.virtualizer = new Virtualizer(this, this.table);
         this.layoutManager = new TableLayoutManager(this);
+        this.layoutManagerNotifier = Observable.getNotifier(this.layoutManager);
+        this.layoutManagerNotifier.subscribe(this);
```

Comment:
I think one of these, probably subscribe, can take the property name to observe. Could maybe filter out some unnecessary notifications since we just care about 1 prop.